packages/components/button/src/Button/Button.tsx

```diff
@@ -82,7 +82,7 @@ function _Button<E extends React.ElementType = typeof BUTTON_DEFAULT_TAG>(
     <>
       {startIcon && iconContent(startIcon)}
       {children && (
-        <Box as="span" display="block" className={styles.buttonContent}>
```

Comment:
Is this fix also related to Workbench fix?

Reply:
@bgutsol Yes it is related. SVGs inside block elements adds extra space "browser behaviour" display `inline-flex` is one way to fix it

Reply:
But we don't pass SVGs as children to the button, we have `startIcon/endIcon` props for that. What exactly you are trying to fix with it?

Reply:
Reverted: Back button in workbench was using button and injecting an icon as a children. Instead I refactored it to use IconButton which uses `flex` 🤷🏼

Reply:
Awesome, thank you 🙌